sdk/storage/Azure.Storage.Blobs/src/Models/PageBlobOpenWriteOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Optional parameters for Page Blob Open Write.
+    /// </summary>
+    public class PageBlobOpenWriteOptions
+    {
+        /// <summary>
+        /// Whether the upload should overwrite any existing blobs.
+        /// The default value is false.
+        /// </summary>
+        public bool Overwrite { get; set; }
+
+        /// <summary>
+        /// The size of the buffer to use.  Default is 1 MB,
+        /// max is 4 MB.  Must be a increment of 512.
```

Comment:
More referencing exposed properties that describe these

Reply:
Same as above.